lib/TabsNew/styles.scss

```diff
@@ -0,0 +1,17 @@
+.tab-drag-line {
```

Comment:
🤔 It is becoming more and more apparent that `tailwind` is good for the generic stuff, but trying to do the nitty gritty stuff (like this) is a pain!

Reply:
perhaps we could use [components](https://tailwindcss.com/docs/extracting-components/) for our more custom fiddly stuff

Reply:
> 🤔 It is becoming more and more apparent that `tailwind` is good for the generic stuff, but trying to do the nitty gritty stuff (like this) is a pain!

100%, sometimes we just need some really nice specific styles!

Reply:
> perhaps we could use [components](https://tailwindcss.com/docs/extracting-components/) for our more custom fiddly stuff

Ooo I'll have a look